index.js

```diff
@@ -47,6 +45,7 @@ low.registerLanguage(
   'brainfuck',
   require('highlight.js/lib/languages/brainfuck')
 )
+low.registerLanguage('c-like', require('highlight.js/lib/languages/c-like'))
```

Comment:
We didn't move this file?  **It's also no longer need to be loaded at all.**  It's built into `c` and `cpp` at buildtime.

Reply:
that change was auto-generated by your scripts. want me to remove that line?
(i just ran `npm it`)

Reply:
I'd ask @lowlight.  We're probably stuck with it until v11 as removing it could break if people are referencing it directly (though shouldn't be)... but technically it's no longer required for proper functioning of `c`, `cpp`, `arduino`, etc... it's only purpose would be if people have hard coded `c-like` somewhere in their code/data.